conf_stress_tests.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+import os
+
+def run_stress_test(
+    malloc_conf,
+    batch_size = 1000,
+    batch_frees = 200,
+    num_threads = 100,
+    num_runs = 10000,
+    malloc_size = 32):
+    # The default setting takes ~1min for each test.
+    env = os.environ.copy()
+    env['MALLOC_CONF'] = malloc_conf
+    process = subprocess.Popen(
+        [
+            './fillFlush',
+            f'--batch_size={batch_size}',
+            f'--batch_frees={batch_frees}',
+            f'--num_threads={num_threads}',
+            f'--num_runs={num_runs}',
+            f'--malloc_size={malloc_size}'
+        ],
+        env=env,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+    stdout, stderr = process.communicate()
+    print(f'MALLOC_CONF={malloc_conf}, '
```

Comment:
Let's split this print, first printing out the malloc conf, saying `Running with MALLOC_CONF=...` before invoking the test, then when it's finished output the return value.

This helps if some test is stuck and we want to separately repro using the current conf